HHS/Head-Start-TTADP
# Update 'Completed' objective status to 'Complete'

## Description of change

This change aims to unify the objective status from 'Completed' and 'Complete' to just 'Complete'.

## How to test

- Look at all code changes and verify all necessary locations have been updated (and nothing extra).
- Run the DB migration, check the DB for only 'Complete' objective status.
- Make sure Goal and Objective Creation from RTR works as it did before.
- Make sure Goal and Objective Creation from AR works as it did before.
- Double check Objective Read-only fields for 'Complete' status.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
